prisma/schema.prisma

```diff
@@ -31,3 +33,75 @@ model Session {
   createdAt DateTime @default(now()) @db.Timestamptz(6)
   updatedAt DateTime @updatedAt @db.Timestamptz(6)
 }
+
+model ArtifactTemplate {
+  id             String          @id @default(uuid()) @db.Uuid
+  title          String
+  fieldTemplates FieldTemplate[]
+  artifacts      Artifact[]
+  user           User?           @relation(fields: [userId], references: [id])
+  userId         String?         @db.Uuid
+  visibility     Visibility
+  createdAt      DateTime        @default(now()) @db.Timestamptz(6)
+  updatedAt      DateTime        @updatedAt @db.Timestamptz(6)
+}
+
+model FieldTemplate {
+  id                 String           @id @default(uuid()) @db.Uuid
+  order              Int              @unique
```

Comment:
This @unique will make it so that there can only ever be one field in the entire database with a given order.